src/jobs/hex_publish.yml

```
@@ -0,0 +1,13 @@
+description: >
+  Publish the package to hex
+
+executor:
+  name: docker_membrane
+  mix_env: dev
+
+steps:
+  - checkout
+  - get_mix_deps
+  - run:
```

I'd add a cache restore step, as the project is compiled to generate docs:
```suggestion
  - get_mix_deps
  - use_build_cache:
      env: dev
  - run:
```
(ensure I haven't messed up that code)